crds/operators.coreos.com_operatorconditions.yaml

```diff
@@ -11,6 +11,8 @@ spec:
     kind: OperatorCondition
     listKind: OperatorConditionList
     plural: operatorconditions
+    shortNames:
+    - condition
```

Comment:
This will be removed when the next version of the CRD is generated. Can you append a shortname annotation to the [go struct](https://github.com/operator-framework/api/blob/15d24f1a8352499de54d5efff97dd8b885fc1359/pkg/operators/v1/operatorcondition_types.go#L30)?

Example: https://github.com/operator-framework/api/blob/15d24f1a8352499de54d5efff97dd8b885fc1359/pkg/operators/v1/operatorgroup_types.go#L61

Reply:
Also, make sure to run `make manifests` in the root of the repository after making this change.

Reply:
@awgreene Sorry for the late reply, got it, thanks very much! Updated it.